pyasic/miners/backends/luxminer.py

```diff
@@ -146,6 +147,22 @@ async def reboot(self) -> bool:
     async def get_config(self) -> MinerConfig:
         return self.config
 
+    async def upgrade_firmware(self) -> dict:
```

Comment:
Would prefer this return a boolean status.

Reply:
I've changed.